lib/spack/llnl/util/link_tree.py

```diff
@@ -47,34 +49,50 @@ class LinkTree(object):
 
     """
 
-    def __init__(self, source_root):
+    def __init__(self, source_root, merge_file=None, check_conflict=None,
```

Comment:
maybe document what values merge_file, check_conflict and remove_file can take. I think you are are looking for functions ... but it should be more obvious than reading the source code

Reply:
@healther thanks for catching that! Those args are no longer needed in this PR. I'll comb through the other functions tomorrow and check for similar issues.